### DISCUSSION TITLE
how to handle if some key of external_json_data contains whitespace

### DISCUSSION THREAD

**Discussion by grimmerk** [pydantic/pydantic#2858] [Question] [answered]
e.g. 
```
external_data = {
    'id': '123',
    'photo view': 3,
    'friends': [1, 2, '3'],
}

user = User(**external_data)
```
Is any way to define `photo view` attribute in User class?

**Comment by samuelcolvin** [comment, marked as answer]
I'm not sure what the question is. I think you want to use `alias`, see the docs.

**Reply by grimmerk** [reply]
Thanks, I will take a look.  

I got a JSON data from some API and want to map it to Python object. But that data has some wired key like `photo view` containing whitespace. I do not know how to define the corresponding attribute. 

```
class User(BaseModel):
    id: int
    friends: List[int] = []
    photo view <- how to define some attritube which contains some whitespace. 
    
external_data = {
    'id': '123',
    'photo view': 3,
    'friends': [1, 2, '3'],
}
user = User(**external_data)
```

**Reply by grimmerk** [reply]
@samuelcolvin `alias` works !! Thank you very much.
